### PR TITLE
Homepage partners section - #3

### LINKED ISSUE #3: 🎨 HomePage Partners Section
#  🎨 Create the partner's section on the HomePage
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that shows/lists all our core business partners. Check the full design below for section placement. Use the assets provided below.

partners-section

Please take note of all the design features

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```

#### Company Logos
VVOB:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/vvob-logo_qyvjbp.jpg
```

KSA:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/Kenya_Scouts_Association_kcoka5.svg
```

farmbetter:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/farmbetter-logo_x2acda.png
```
WWF:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/wwf-4-logo-png-transparent_tbkfcq.png
```

Enabel:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380405/elewa-group-website/client-logos/Enabel-logo_afiel7.png
```

Spotlights:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380412/elewa-group-website/client-logos/Spotlights_7_pplnvm.jpg
```

### Technical Analysis
This section should be present in the `index.html` file

### Acceptance Criteria
1. When rendered the component is pixel-perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by jessemutua**
# Description

I added the provided images in HTML and styled them making them responsive for different screen sizes.

Fixes #3

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screenshot from 2023-10-03 16-46-54

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jessemutua** [comment]
@LemmyMwaura

File: index.html [hunk 1]
```diff
@@ -2,18 +2,31 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap">
 
-  <script defer type="module" src="dist\out-tsc\index.js"></script>
+
+    <script defer type="module" src="dist\out-tsc\index.js"></script>
 </head>
 
 <body>
-  <div>Welcome</div>
+    <div class="container">
```
Comment (JAPHETHNYARANGA) [inline index.html]: what is the purpose of this welcome

File: src/css/style.css [hunk 2]
```diff
@@ -1,5 +1,134 @@
...
+        max-width: 100%;
+    }
+    /* Styling for the .container on small screens */
+    .container {
+        display: flex;
+        justify-content: center;
+        /* Horizontally center the content */
+        align-items: center;
+        /* Vertically center the content */
+        height: 100vh;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: it will be a page, not a section , with this property

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
please check the background color codes, and how will the section be used if it is a whole page instead of a section. Media queries are used to make the page responsive, the application may need styles before making the page responsive, i cannot see this. Please address this. Thank you

**Comment by jessemutua** [comment]
@LemmyMwaura

**Comment by jessemutua** [comment]
@JAPHETHNYARANGA , i have reviewed and resolved the issues raised

**Comment by jessemutua** [comment]
@LemmyMwaura , I reviewed the issues raised

**Comment by jessemutua** [comment]
@LemmyMwaura , I reviewed the issues raised

**Comment by JAPHETHNYARANGA** [comment]
good work but closing this pr as the issue is done ,please pick another issue. Thank you

**Pull request closed without merging**